procrastinate/utils.py

```diff
@@ -329,3 +329,17 @@ def queues_display(queues: Iterable[str] | None) -> str:
         return f"queues {', '.join(queues)}"
     else:
         return "all queues"
+
+
+def ellipsize_middle(
```

Comment:
I like this function, and it seems like the perfect candidate for a `parametrize` set of unit tests :)

In particular, I like that this implementation works well around 100 chars (if we look at what happens char by char, there's no moment where we add 1 char and we end up with ellipsis and with the whole string being less than 100 chars). That said, I felt the need to test it to be 100% sure of that, so it would be awesome if the tests were included :D

I specifically liked a lot that all the important configuration is passed as parameters.